cli/js/os.ts

```diff
@@ -129,193 +129,106 @@ export function start(preserveDenoNamespace = true, source?: string): Start {
  * | macOS   | `$HOME/Library/Fonts`                                | /Users/Alice/Library/Fonts     |
  * | Windows | –                                                    | –                              |
- */
-export function fontDir(): string {
-  return sendSync(dispatch.OP_GET_DIR, { name: "font" });
-}
-
-/**
- * Returns the current user's picture directory.
- * If the directory does not exist, an exception is thrown
- * Requires the `--allow-env` flag.
+ *
+ * "picture"
  * |Platform | Value                 | Example                 |
  * | ------- | --------------------- | ----------------------- |
  * | Linux   | `XDG_PICTURES_DIR`    | /home/alice/Pictures    |
  * | macOS   | `$HOME`/Pictures      | /Users/Alice/Pictures   |
  * | Windows | `{FOLDERID_Pictures}` | C:\Users\Alice\Pictures |
- */
-export function pictureDir(): string {
-  return sendSync(dispatch.OP_GET_DIR, { name: "picture" });
-}
-
-/**
- * Returns the current user's public directory.
- * If the directory does not exist, an exception is thrown
- * Requires the `--allow-env` flag.
+ *
+ * "public"
  * |Platform | Value                 | Example             |
  * | ------- | --------------------- | ------------------- |
  * | Linux   | `XDG_PUBLICSHARE_DIR` | /home/alice/Public  |
  * | macOS   | `$HOME`/Public        | /Users/Alice/Public |
  * | Windows | `{FOLDERID_Public}`   | C:\Users\Public     |
- */
-export function publicDir(): string {
-  return sendSync(dispatch.OP_GET_DIR, { name: "public" });
-}
-
-/**
- * Returns the current user's template directory.
- * If the directory does not exist, an exception is thrown
- * Requires the `--allow-env` flag.
+ *
+ * "template"
  * |Platform | Value                  | Example                                                    |
  * | ------- | ---------------------- | ---------------------------------------------------------- |
  * | Linux   | `XDG_TEMPLATES_DIR`    | /home/alice/Templates                                      |
  * | macOS   | –                      | –                                                          |
  * | Windows | `{FOLDERID_Templates}` | C:\Users\Alice\AppData\Roaming\Microsoft\Windows\Templates |
- */
-export function templateDir(): string {
-  return sendSync(dispatch.OP_GET_DIR, { name: "template" });
-}
-
-/**
- * Returns the current user's video directory.
- * If the directory does not exist, an exception is thrown
- * Requires the `--allow-env` flag.
+ *
+ * "video"
  * |Platform | Value               | Example               |
  * | ------- | ------------------- | --------------------- |
  * | Linux   | `XDG_VIDEOS_DIR`    | /home/alice/Videos    |
  * | macOS   | `$HOME`/Movies      | /Users/Alice/Movies   |
  * | Windows | `{FOLDERID_Videos}` | C:\Users\Alice\Videos |
  */
-export function videoDir(): string {
-  return sendSync(dispatch.OP_GET_DIR, { name: "video" });
+export function dir(name: string): string {
```

Comment:
Maybe it would be better to write like this

```ts
type dirName = "home" | "cache" | "config";
export function dir(name: dirName): string {
  //
}
```

Reply:
done